cardboard/settings.py

```diff
@@ -146,14 +146,19 @@
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.2/howto/static-files/
+# https://whitenoise.readthedocs.io/en/latest/django.html
 
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 STATIC_URL = "/static/"
 STATICFILES_DIRS = [
     os.path.join(BASE_DIR, "cardboard/static"),
     os.path.join(BASE_DIR, "hunts/static"),
 ]
-STATICFILES_STORAGE = "whitenoise.django.CompressedManifestStaticFilesStorage"
+STORAGE = {
+    "staticfiles": {
+        "BACKEND": "whitenoise.storage.CompressedManifestStaticFilesStorage",
+    }
+}
```

Comment:
Is this change required? Or is it basically the "new way" to do things? (in which case, seems reasonable to switch)

Reply:
I tested with this syntax vs the other and it seems necessary. According to the docs, this is needed for Django 4.2+

Reply:
Sounds good. FWIW, I think Django is still 4.1 in the poetry.lock file.